lemarche/siaes/models.py

```diff
@@ -121,6 +145,9 @@ def annotate_with_user_favorite_list_ids(self, user):
             in_user_favorite_list_ids=ArrayAgg("favorite_lists__pk", filter=Q(favorite_lists__user=user))
         )
 
+    def filter_with_email(self):
```

Comment:
```suggestion
    def has_contact_email(self):
```
pour être homogène avec les autres queryset existants (plus haut)